app/controllers/hub/documents_controller.rb

```diff
@@ -20,35 +20,36 @@ def show
       redirect_to transient_storage_url(@document.upload.blob)
     end
 
+    def new; end
+
     def edit; end
 
     def create
       document_params[:upload].each do |file_upload|
         Document.create!(
```

Comment:
Does it make sense to allow for uploading multiple files with the same display name? Maybe we should limit this upload to uploading one file at a time!

Reply:
Yes, we are in talks with Yvonne about that now. It's not perfect, but figured a good interim step would be to have all files named the same (unless no name given) rather than take away the multi upload functionality.

Waiting on design for how to handle this.

Reply:
After talking with Yvonne, we are not going to remove multi upload functionality - we do have a story in the works to make this process smoother though. In the mean time if you want to multi upload, you'll have to go in and edit the features later.